content/pages/YakShaver/try-it-now.json

```
@@ -44,9 +43,9 @@
           "title": "3. Start Recording",
           "description": "Click on the YakShaver Logo ![](/TryItNow/yakshaver.svg) at the top of  your browser window, then click Start Recording.\n",
           "image": {
-            "imgSrc": "/TryItNow/yakshaver-this-tab.png",
-            "imgWidth": 480,
-            "imgHeight": 336
+            "imgSrc": "/TryItNow/yakshaver-this-tab2.png",
```

Why 2? isn't the file replaced?
```suggestion
            "imgSrc": "/TryItNow/yakshaver-this-tab.png",
```

---

It's a bug with Tina cloud, I'm not currently able to replace the image in the editor and have it appear on the page.